pkgs/pkgs-lib/formats.nix

```diff
@@ -109,18 +109,21 @@ rec {
       singleIniAtom = nullOr (oneOf [ bool int float str ]) // {
         description = "INI atom (null, bool, int, float or string)";
       };
-      iniAtom = { listsAsDuplicateKeys, listToValue }:
+      iniAtom = { listsAsDuplicateKeys, listToValue, mergeAsList }:
```

Comment:
I think something like `atomsCoercedToLists` would be a better name since this this won't just affect merging. With this change, options queried through the `config` param will now return atoms instead of lists for non-list settings.